lib/teams/aws-batch/aws-batch-on-eks-team.ts

```diff
@@ -0,0 +1,167 @@
+import { assertEC2NodeGroup } from "../../cluster-providers";
+import { ClusterInfo, Values } from "../../spi";
+import { ApplicationTeam, TeamProps } from "../team";
+import { KubectlProvider, ManifestDeployment } from "../../addons/helm-addon/kubectl-provider";
+import { loadYaml, readYamlDocument } from "../../utils/yaml-utils";
+import { Construct } from "constructs";
```

Comment:
Please address github action warning `Construct` is defined but never used as well as other warnings elsewhere.